### ISSUE TITLE
AccessibleValueHandler.endInteractionUtterance improvements

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#637]
I got here from https://github.com/phetsims/ratio-and-proportion/issues/214, and feel like we could do a bit more here. In that issue @terracoda said:

>@zepumph, just noting that when using the sliders on VoiceOver, all the context responses containing distance from other hand and proximity to ratio pile up easily, too.

That particular comment doesn't apply here, but it got me thinking on the following updates:

* rename to `interactionEndUtterance`
* make it a `ValueChangeUtterance` (maybe?)
* allow options to be passed into the utterance so that it can be tweaked it needed. I wouldn't want to have to use my own utterance just because I want to tweak the alertStableDelay or something. 

@jessegreenberg can I run these changes by you during today's meeting?

**Comment by jessegreenberg** [comment]
Discussed with @zepumph - 

We reviewed usages of ValueChangeUtterance and determined that it should be removed and replaced with AccessibleValueHandler.a11yCreateValueChangeAlert. Almost all of the existing usages of ValueChangeUtterance should be straight forward except for ones in gravity-force-lab-basics and in molarity.

In gravity-force-lab-basics, the usage around the `secondaryPositionChangedListener` will need to remain as is because it is used in many more places than just response to an input element change.

In molarityAlertManagery we hope to return `null` from `a11yCreateValueChangeAlert` when we are using an utterance other than the current usage of valueChangeUtterance, then send our own utterance to the queue in that case. Otherwise, we will use AccessibleValueHandler.a11yCreateValueChangeAlert.

We will then rename ValueChangeUtterance to ContextResponseUtterance to be in line with terminology used by the whole team.

We also noticed that `AccessibleValueHandler.setUtteranceAndAlert();` (which eventually calls `setUtteranceAndAlert` is only used with alternative input. We would like to make sure that `setUtteranceAndAlert` gets used at the end of mouse/touch input as well.

**Comment by zepumph** [comment]
>We also noticed that AccessibleValueHandler.setUtteranceAndAlert(); (which eventually calls setUtteranceAndAlert is only used with alternative input. We would like to make sure that setUtteranceAndAlert gets used at the end of mouse/touch up as well.

After thinking this through a bit more, I don't feel like this is correct. It assumes that the alt input listeners are added to the same Node as the mouse/touch listeners. This is often not true. Furthermore, `up` is the most granular that we can be in AccessibleValueHandler, but we could do a lot better by calling the same function in DragListener.end(). I think that a more appropriate way to handle this would be to rename `AccessibleValueHandler.setUtteranceAndAlert()` to `AccessibleValueHandler.alertContextResponse()`, and to make that a public function.

That feels first to me, then we can worry about ValueChangeUtterance changes. That is also the blocking piece for an important issue for interviews over in https://github.com/phetsims/ratio-and-proportion/issues/220

**Comment by zepumph** [comment]
I did some renaming here, and created https://github.com/phetsims/scenery-phet/issues/638. The main thing left for this issue is to discuss the two "arbitrary values" that control the alert stable delay of utterance. I feel like at the very least these should be options, since we are here right now. They both feel like they may need to be updated based on the usage, and how long the object response (aria-valuetext) is.

**Comment by zepumph** [comment]
I'm a bit nervous about these values for everything. I think over in https://github.com/phetsims/scenery-phet/issues/638 we may need to use the options. Then see if there are any consistent winners for these values. 

One potentially different algorithm is to have a static delay time for the first utterance, and the a second time (perhaps 200ms longer) for each additional time, but to never multiply. For now though I think this issue is ready for closing. I'll do the rest over in https://github.com/phetsims/scenery-phet/issues/638.

**Comment by zepumph** [comment]
In https://github.com/phetsims/sun/commit/0957106f1a521e57323af4c8ccd900fd69a0c214 @terracoda and I decided during a meeting that 1500 was worth trying out. I think it is working well now. Moving on to https://github.com/phetsims/scenery-phet/issues/638. Closing
